core/src/main/java/org/elasticsearch/index/query/SimpleQueryStringParser.java

```diff
@@ -25,7 +25,6 @@
 import org.elasticsearch.common.settings.Settings;
 import org.elasticsearch.common.xcontent.XContentParser;
 import org.elasticsearch.index.mapper.MappedFieldType;
-import org.elasticsearch.index.query.SimpleQueryStringBuilder.Operator;
 
 import java.io.IOException;
 import java.util.HashMap;
```

Comment:
Oh nice. Is it ok to leave the IllegalArgumentException when parsing the text fails or should that be changed to the QueryParsingException we had before?